packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -510,13 +557,36 @@ export async function emitCode(context: vscode.ExtensionContext, uri: vscode.Uri
           picked: true,
           fromConfig: true,
           package: e,
+          language: e,
+          emitterKind: EmitterKind.Unknown,
           kind: vscode.QuickPickItemKind.Default,
         };
       }
     });
+    const multipleSelectionSeparatorItem = {
```

Comment:
do you think its valuable to have a multi select vs just have emit all? This would get rid of 1 extra interation

Reply:
@timotheeguerin  Customers may need to select several existing emitters to regenerate, rather than selecting all emitters, which we believe is a common scenario. Therefore, we have defined a quick pick item for `multiple-selection` instead of an `all` option. We decided not to provide an additional all quick pick item because it would make the UI too complex with too many types of quick pick items, leading to redundancy. Although selecting all emitters from multiple-selection may require extra interaction, we consider it acceptable.

Reply:
I think this is something we need to go talk to the designer team. The experience is clumsy right now, I need to select so many things. Basically don't feel like there is much point in this whole UI as it is.
It doesn't need to block this pr but I don't think we are in a good spot.

Reply:
Thanks @timotheeguerin We will bring this to designer team, and refine the UI later.